ui/ducks/index.js

```diff
@@ -22,6 +23,7 @@ export default combineReducers({
   history: historyReducer,
   send: sendReducer,
   confirm: confirmReducer,
+  confirmAlerts: confirmAlertsReducer,
```

Comment:
This was the best name I could think of as the `alerts` directory was already in use.

My thinking is `confirm` doesn't couple it to confirmations but just means the alerts can be confirmed.

`acknowledgeAlerts` seemed too long 😄

Reply:
yeah at first glance, I thought it was related to confirmations but after start working with the alerts I started seeing more relation with the ack and confirm than anything with confirmations.
I also tried some alternatives, but nothing shined.